CIScripts/Tests/TestConfigurationUtils.ps1

```diff
@@ -161,6 +161,14 @@ function Test-IsDockerDriverEnabled {
     return Test-IsProcessRunning -Session $Session -ProcessName "contrail-windows-docker"
 }
 
+function Test-IsDockerDriverListening {
```

Comment:
In my opinion there still might be a race, since docker plugin is registered later in docker driver code. If we would like to eliminate all possible race conditions we should verify if docker plugin is registered.

Reply:
Added a check